.github/workflows/ci.yml

```diff
@@ -67,7 +67,7 @@ jobs:
             chmod -R 777 ./zio-ftp/ftp-home/sftp/home/foo/dir1
       - name: Test 2.x
         if: ${{ !startsWith(matrix.scala, '3.') }}
-        run: ./sbt ++${{ matrix.scala }} test
+        run: ./sbt ++${{ matrix.scala }} zio-ftp/test
```

Comment:
i dont understand why we need to create a subfolder zio-ftp.
Please revert it.
Since it is a single module, we dont need to create this layer

When you are cloning the repo, you will have zio-ftp/zio/ftp

Reply:
@regis-leray 
This PR contains two modules `zio-ftp` and the `docs` module. So it is not a single module. If you wish, I can revert that. But would like to know what is your concern about these changes.